core/integration/client_tests.go

```diff
@@ -67,23 +67,34 @@ LOA+tLe/MbwZ69SRdG6Rx92f9tbC6dz7UVsyI7vIjS+961sELA6FeR91lA==
 -----END PUBLIC KEY-----`
 )
 
+func runBatchAndPublish(ctx context.Context, dirID string, mn, mx int32, block bool, env *Env) error {
```

Comment:
I like to put env right after context for consistency

Reply:
Done.